hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -172,6 +275,9 @@ interface IExpandedRowState {
 interface IMissingContainersState {
   loading: boolean;
   dataSource: IMissingContainerResponse[];
+  udataSource: IContainerResponse[];
+  odataSource: IContainerResponse[];
+  mdataSource: IContainerResponse[];
```

Comment:
1. Add a comment clarifying that `u` stands for `underReplicated` here; or
2. Use full name here?
```suggestion
  udataSource: IContainerResponse[];
  odataSource: IContainerResponse[];
  mdataSource: IContainerResponse[];
```